src/components/Navigation/Navigation.jsx

```diff
@@ -42,22 +43,32 @@ export default function Navigation(){
 	const ariaOpen    = open.toString();
 
 	return (
-		<nav>
+		<nav className={ s.wrapper }>
 			<button 
+				className={ s.button }
 				role="switch"
 				aria-checked={ ariaOpen }
 				aria-controls="navigation__links"
 				onClick={ toggleOpen }
 			>
-				<span>
-					Menu
-				</span>
-				<span>
-					Close
+				<span className={ s.labelContainer }>
+					<span 
+						className={`${s.label} ${s.open}`}
+						aria-hidden="true"
+					>
+						menu
+					</span>
+					<span 
+						className={`${s.label} ${s.close}`}
+						aria-hidden="false"
+					>
+						close
+					</span>
 				</span>
 			</button>
```

Comment:
Can we move this into its own `<NavToggle>` component pretty-please? 🙏  Just create a new `Navigation` context and have the new component dispatch a `OPEN_NAVIGATION` action or what-have you 😋

Reply:
OK, reducer added to navigation which the new NavToggle uses to change the open boolean.
Still using a state in Navigation as it needs to keep aria-expanded updated, it does this by passing a callback function to NavToggle.

Also did styling for Socials. Pretty basic but it's there 😄